### DISCUSSION TITLE
Should we pick a different name for the project?

### DISCUSSION THREAD

**Discussion by kirtangajjar** [sorry-cypress/sorry-cypress#283] [General]
Hey there :wave: 

Absolutely love the work done here. Just had a small suggestion, the current name sometimes scares people a little bit(happened today with my manager), gives a hacky-side-project feel and seems a little risky as cypress.io could jinx this project because of trademark/copyright violations, especially since there is also a cloud based paid service.

So just thought to put it out here. LMK your thoughts :smiley:

**Comment by agoldis** [comment]
Good point. It is indeed a provocative name that was picked at random just when the project started. Having a more conventional term would help.

I am moving this issue to a discussion board. Mid / long term we'll probably change the name. I am also looking for legal advice on that manner.

**Comment by agoldis** [comment]
Hello visitor, if you happen to have a good understanding of the legal aspect for using "sorry-cypress" name, please reach to me and share your wisdom!

**Comment by dvngnt** [comment]
@agoldis I suggest reading the [terms of service](https://www.cypress.io/terms-of-use) 

The name should probably be changed since they have the rights to that name, and adding a hyphen probably isn't enough. 

Also, I suggest removing all images from the site and documentation page that you don't have rights to use https://docs.sorry-cypress.dev/cypress-agent/parallelization-guide since this project is now for-profit.

**Comment by agoldis** [comment]
I will be using different names for sensitive/commercial parts of the project. 

It seems like the GitHub repo, and the project itself can stay as is.

- There's no problem using the `cypress tests` term because there's no other way to describe this thing
- There's no problem using the "Cypress" term to refer to the company because that's the company name, and there's no other way to call it
- There's a potential problem with people confused by the name and affiliating it with the original company, but we state clearly that it's an alternative, and we are not affiliated.

@dvngnt 

TL;DR it's quite confusing

I see your concern, we are not-for-profit yet, but I am planning to brand it differently when the time comes. 
The Term of service you're referencing is www.cypress.io website terms of use, i.e., if you are to use the website, you'd comply with the terms. 
There's an open-source, MIT licensed, open-source software hosted on Github, called `cypress`, it has different terms; also, there'sCypress Dashboard` service provided by Cypress Inc., which is a commercial product...

**Reply by kirtangajjar** [reply]
Although the GH repo can remain as it is, but my question is why to have a provocative name for a project? It's not a troll work or done just to actively harm cypress in any way. You've done really good work and it's a quiet a decent project. I would rather have a name like `OpenCypressDashboard` or `CyDashboard` or anything else that's not provocative. 

IMO, these names would do justice to the project, would more accurately represent the project and also help position it as a mature product.

**Reply by agoldis** [reply]
@kirtangajjar 

Thanks for your feedback. It's good because I can see now how others perceive it. 

At this stage, it's mostly about:
- discoverability  - the project is known as "sorry-cypress" + there're some online posts that use the name + some SEO ranking
- my time - I don't want to spend it just now. I'd rather invest differently at this stage. mid / long term I'd rather change it

I agree that might be considered provocative. 2 years ago, I randomly picked the name while committing a code to run my tests in parallel late at night. That's just as simple as it is :)

**Reply by agoldis** [reply]
> It seems like the GitHub repo, and the project itself can stay as is.

The original comment was about Trademarks violation - not the ethical aspect of the name
